app/services/search_indexer.rb

```diff
@@ -189,7 +181,7 @@ def self.scrub(html, strip_diacritics: false)
 
       me = new(strip_diacritics: strip_diacritics)
       Nokogiri::HTML::SAX::Parser.new(me).parse("<div>#{html}</div>")
-      me.scrubbed
+      me.scrubbed.squish
```

Comment:
```
[1] pry(main)> "test\n   \ntest".squish
=> "test test"
```
Do we really want to eat up every newline? What impact will this have on excerpts ?

Reply:
Good point. I just wanted to remove consecutive white spaces (but leave new lines).

Reply:
Actually, search results were always squished https://github.com/discourse/discourse/blob/aca195e4a7fb760009e3e57e2a884349ed56cf08/lib/search/grouped_search_results.rb#L66